fftest/vars.go

```diff
@@ -27,6 +27,42 @@ func Pair() (*flag.FlagSet, *Vars) {
 	return fs, &v
 }
 
+// EmptyValsFS returns a predefined flag set, and a predefined set of variables that
+// have been registered into it.
+// Custom Test Flagset creation function for the Empty (YAML) Values parsing, using more
+// specific flag names.
```

Comment:
Why is this necessary?

Reply:
> Why is this necessary?

See my response here -> https://github.com/peterbourgon/ff/pull/82#discussion_r660864051.

It is not technically _necessary_. 

I chose to create `EmptyValsFS()` to use more descriptive field names in this function and in the `empty_basic_vals.yaml` for this test case. 

I thought that might be helpful on combination with function TestParserAllTypesEmptyVals() to more clearly self-document this new/additional functionality of the YAML parser that I am proposing in this PR.

Reply:
> I do not think having field names s, i, b, d, f in the test fixture function or in the empty_basic_values.yaml file contributes to that unerstanding.

I'm confused. There is no difference between the FlagSets returned by this function and Pair, except for the names of the flags — in both cases, the default value is the zero value for the associated type. Is your claim that the name of the flag is meaningful by itself? If so, I reject the claim 🙃 and request that you remove this helper in favor of Pair. If there is another technical reason, please correct my understanding!